runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/fn/control/BeamFnMapTaskExecutor.java

```diff
@@ -351,7 +398,115 @@ void updateProgress() {
       }
     }
 
-    private void updateMetrics(BeamFnApi.Metrics metrics) {
+    // Keeping this as static class for this iteration. Will extract to separate file and generalize
```

Comment:
This first sentence comment seems to refer specifically to the PR. I suggest removing it as the rest of the comment is sufficiently descriptive.